### DISCUSSION TITLE
High Availability for KVM VM

### DISCUSSION THREAD

**Discussion by HtainL** [actions/virtual-environments#8118] [General questions]
Hi Sir,
I would like to ask a question that i installed ubuntu 22.04 in two dell r630 server. Two servers was connect with sharestorage mountpoint at /etc/libvirt/qemu and /var/lib/libvirt/images. So, two servers can access vm.xml and vm.qcow2. When one of my ubuntu server was failed, i want to auto run all my vm to another server.
I configured with pacemaker, corosync and pcs and i shutdown my server but it didn't auto run in another server. Please help me to provide me how to do it? 

Please see the below command that i created in pacemaker.

"pcs resource create Cluster VirtualDomain hypervisor="qemu:///system" config="/etc/libvirt/qemu/ubuntu22.04.xml" migration_transport=ssh op start timeout="10s" op stop timeout="10s" op monitor  timeout="10" interval="10"  meta allow-migrate="true" priority="100" op migrate_from interval="0" timeout="10s" op migrate_to interval="0" timeout="10" --group root"

**Comment by vpolikarpov-akvelon** [comment]
Hi @HtainL.

We'd be happy to assist you with your query, but it appears that your question might be better suited for a more specific repository or forum that focuses on KVM and QEMU or Ubuntu 22.04.

This repository primarily deals with GitHub Actions runner images and related topics. While one of the runner images is based on Ubuntu 22.04, the repository's main focus is not on general Ubuntu or virtualization configuration.

For inquiries related to KVM and QEMU configuration on Ubuntu 22.04, I recommend visiting [Official Ubuntu Support Forums](https://ubuntuforums.org/), GitHub Repositories dedicated to KVM and QEMU or even Stack Overflow. By posting your question in the appropriate forums or repositories, you'll have a better chance of receiving targeted and expert assistance on your query.

If you have any GitHub Actions or runner-related questions in the future, feel free to return to this repository, and we'll be more than happy to help. Thank you for your understanding, and I wish you the best of luck with your virtualization endeavors.
